File: Domovoy/Models/Informer/InformMeter.cs
namespace Domovoy.Models;

public class InformMeter
{
    public int Id { get; set; }
    public Informer Informer { get; set; }
    public int InformertId { get; set; }

    public int Count { get; set; }
    /// <summary>
    /// Для различных проверок
    /// </summary>
    public int LastInformedCount { get; set; }
}

File: Domovoy/ClientApp/src/api/models/InformMeter.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Informer } from './Informer';

export type InformMeter = {
    id?: number;
    informer?: Informer;
    informertId?: number;
    count?: number;
    /**
     * Для различных проверок
     */
    lastInformedCount?: number;
};